python/arcticdb/version_store/_normalization.py

```diff
@@ -258,7 +258,7 @@ def _to_tz_timestamp(dt):
 
 def _from_tz_timestamp(ts, tz):
     # type: (int, Optional[str])->(datetime.datetime)
-    return pd.Timestamp(ts).tz_localize(tz).to_pydatetime()
+    return pd.Timestamp(ts).tz_localize(tz).to_pydatetime(warn=False)
```

Comment:
https://pandas.pydata.org/docs/reference/api/pandas.to_datetime.html

I don't see that argument documented there?

Reply:
Wrong method:
https://pandas.pydata.org/docs/reference/api/pandas.Timestamp.to_pydatetime.html

Reply:
🙄 

Cool, thanks